### PR TITLE
Updating service and associated doc

### PR DESCRIPTION
**Pull request by mbcoder**
### Description

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Review by alan-edi — APPROVED** [review]
@mbcoder LGTM

File: tiled_layers/web-tiled-layer/src/main/java/com/esri/samples/web_tiled_layer/WebTiledLayerSample.java [hunk 1]
```diff
@@ -68,10 +68,9 @@ public void start(Stage stage) {
           // set the web tiled layer as the map's basemap
           map.setBasemap(new Basemap(webTiledLayer));
           // set custom attribution on the layer
-          webTiledLayer.setAttribution("Map tiles by <a href=\"http://stamen.com/\">Stamen Design</a>, " +
-              "under <a href=\"http://creativecommons.org/licenses/by/3.0\">CC BY 3.0</a>. " +
-              "Data by <a href=\"http://openstreetmap.org/\">OpenStreetMap</a>, " +
-              "under <a href=\"http://creativecommons.org/licenses/by-sa/3.0\">CC BY SA</a>.");
+          webTiledLayer.setAttribution("Map tiles by ArcGIS Living Atlas of the World" +
+              ", under the Esri Master License Agreement." +
```
Comment (D-R-Smith) [inline tiled_layers/web-tiled-layer/src/main/java/com/esri/samples/web_tiled_layer/WebTiledLayerSample.java]: You might want to add a space between the `.` and `Data`.

**Review by D-R-Smith — APPROVED** [review]
@mbcoder looks fine to me

File: tiled_layers/web-tiled-layer/src/main/java/com/esri/samples/web_tiled_layer/WebTiledLayerSample.java (hunk 1 above)
Comment (mbcoder) [reply tiled_layers/web-tiled-layer/src/main/java/com/esri/samples/web_tiled_layer/WebTiledLayerSample.java]: Yes fair point, I'll add that

**Pull request merged**